merino/configs/default.toml

```
@@ -54,6 +54,15 @@ env = "dev"
 # A setting for the tracing sample rate. Should be a float in range [0, 1.0].
 traces_sample_rate = 0.1
 
+[default.providers.accuweather]
+enabled_by_default = false
+score = 0.3
+api_base_url = "https://apidev.accuweather.com"
+api_param_key = "apikey"
```

```suggestion
api_param_key = ""
```
It'll be easier to validate this configuration with an empty string.

---

Looking at the code below, looks like `api_param_key` is not used to store the API key to Accuweather, correct? We can add a new configuration `api_key` and set its default to `""`.  We will override it with an environment variable (`MERINO_PROVIDERS__ACCUWEATHER__API_KEY`) in production.

---

Right, it's the name of the query param that should be set to the key. (It looks like the provider name has to be specified in lowercase as part of the environment variable, like `MERINO_PROVIDERS__accuweather__API_KEY`.)

---

I'll come up with better names for these.